tests/OpenToolkit.Tests/Assertions.fs

```diff
@@ -51,9 +48,13 @@ type internal Assert =
 
     static member ApproximatelyEquivalent(a : float32,b : float32) =
         if not <| approxEq a b then raise <| new Xunit.Sdk.EqualException(a,b)
+        
+    static member ApproximatelyEquivalent(a : Color4<'T>, b : Color4<'T>, epsilon:float32) =
```

Comment:
Im very picky here again. Probably a `Format Document` will fix already.

Please insert space infront and after `:` at `epsilon:float32`